apps/use-after-free/Main.hs

```diff
@@ -0,0 +1,24 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE StaticPointers #-}
+{-# LANGUAGE TypeApplications #-}
+
+module Main where
```

Comment:
This program can use a comment explaining its purpose and the expected behavior.